src/templates/next/components/TableOfContents/TableOfContents.tsx

```diff
@@ -0,0 +1,22 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { TableOfContentsProps } from "~/common"
+
+const TableOfContents = ({ items }: TableOfContentsProps) => {
+  return (
+    <div className="flex flex-col gap-3 pl-5 border-l-4 border-content w-[18.75rem]">
```

Comment:
should we auto collapse on mobile views? might be that the toc takes up a huge amount of space prior to content. (can check w/ design, also ok if we don't)

Reply:
yea, we are still showing on mobile for now